samples/net/wifi/overlay-nrf70.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_WIFI_NRF70=y
+CONFIG_WIFI_NM_WPA_SUPPLICANT=y
+CONFIG_NET_L2_ETHERNET=y
+# Adjust memories as per nRF70 + sample configuration
+CONFIG_MAIN_STACK_SIZE=5200
+CONFIG_SHELL_STACK_SIZE=5200
+CONFIG_HEAP_MEM_POOL_SIZE=130000
```

Comment:
Please remove the `CONFIG_HEAP_MEM_POOL_SIZE` as this is now handled automatically